charts/eap-xp5/values.yaml

```diff
@@ -32,14 +32,14 @@ deploy:
     httpGet:
       path: /health/live
       port: admin
-    initialDelaySeconds: 60
+    initialDelaySeconds: 10
```

Comment:
`initialDelaySeconds` of the `livenessProbe` should be removed as a `startupProbe` is defined

Reply:
Ok, removed.